src/cascadia/TerminalSettingsModel/ActionArgs.cpp

```diff
@@ -386,4 +387,22 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         }
         return RS_(L"ScrollDownCommandKey");
     }
+
+    winrt::hstring MoveTabArgs::GenerateName() const
+    {
+        winrt::hstring directionString;
+        switch (_Direction)
+        {
+        case MoveTabDirection::Forward:
+            directionString = RS_(L"MoveTabDirectionForward");
+            break;
+        case MoveTabDirection::Backward:
+            directionString = RS_(L"MoveTabDirectionBackward");
+            break;
+        }
+        return winrt::hstring{
+            fmt::format(std::wstring_view(RS_(L"MoveTabCommandKey")),
+                        directionString)
+        };

```

Comment:
It might actually be better to just have two strings, one for "Move tab forward" and another for "Move tab backward". I'm a little worried that there might be a language out there where formatting the string out of two _separately_ localized parts like this might not work the same as localizing the entire string.

Reply:
I can change this, however this is the approach we have with other directions like resize pane, move focus, etc.

Reply:
Oh heck, we sure do don't we. Must not be that much of an issue then!

Reply:
This is the concern I raised when we started piecing commands together out of duct tape and glue. It is a problem. It's also _not_ a problem because apparently people want to match commands in english.